components/spi/kustomization.yaml

```diff
@@ -1,16 +1,16 @@
 resources:
-  - https://github.com/redhat-appstudio/service-provider-integration-operator/config/default?ref=ccc1d8a0b400aa3e47c10fa14040674252d3ae3d
-
+  - https://github.com/sparkoo/service-provider-integration-operator/config/default?ref=c36682a4fd7dbac2e7303637e0cc4c5ff958cfd4
+  - oauth_route.yaml
 
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 images:
   - name:  quay.io/redhat-appstudio/service-provider-integration-operator
     newName: quay.io/redhat-appstudio/service-provider-integration-operator
-    newTag: v0.2.0
+    newTag: v0.2.1
   - name: quay.io/redhat-appstudio/service-provider-integration-oauth
     newName: quay.io/redhat-appstudio/service-provider-integration-oauth
-    newTag: v0.2.0
+    newTag: v0.2.1
```

Comment:
https://quay.io/repository/redhat-appstudio/service-provider-integration-oauth?tab=tags the image is tagged `0.2.1` without `v`

Reply:
fixed